### ISSUE TITLE
Add option to disable vibration in Android

### ISSUE THREAD

**Issue by thranur** [Card-Forge/forge#5256]
**Is your feature request related to a problem? Please describe.**
Some phones have a very noisy vibration motor, such that even in completely silent mode, the long-duration vibration when losing life points can be really noticeable. Unfortunately Android does not offer a way to *completely* disable vibration in such cases (even when the phone is set to silent mode).

**Describe the solution you'd like**
A checkbox, next to the sound effects and music volume controls in the options, that allows disabling vibration when losing life points. Or, alternatively, such checkbox could disable *all* vibrations from the game.

**Describe alternatives you've considered**
- Preventing an Android app from vibrating entirely: not currently possible on Android, as far as I understand
- Not losing life points to avoid vibration: not always possible despite best efforts

**Additional context**
The other vibration modes in Forge (e.g. long-pressing a card to zoom on it) are short and less conspicuous. But since the life points vibration length depends on the amount of life points lost, it can lead to multi-second vibrations, which can bother nearby people (roommates, in public transit, at the library, etc). There are several reports about Android apps (namely games) doing so, and no universal solution is provided by Google. I did find a mention of using ADB to remove vibration permission from an app, but it's extremely cumbersome and I'm not sure it actually works.

**Comment by tool4ever** [comment]
![image](https://github.com/Card-Forge/forge/assets/8506892/97c01071-0250-4fc9-a3bb-4a9a3b47c54b)

??

**Comment by thranur** [comment]
Thanks a lot!

I just found out about these in the Classic Settings menu. Until now, I had *never* tried the Classic mode, I only play on Adventure mode. And so the only Settings menu I'd ever seen was this one:

![scren1](https://github.com/Card-Forge/forge/assets/33481653/48e7c9e8-adcd-4e3f-a57b-78dc6d20ea66) 
![scren2](https://github.com/Card-Forge/forge/assets/33481653/431484f4-a9dc-4906-94c7-276258d8e221)

Because it had a music and sound volume, I didn't think of looking for it somewhere else, I assumed the Adventure mode was "self-contained". And because of the large loading times and frequent crashes due to lack of memory, I refrained from simply clicking "Classic"...

Thank you for that, and sorry for the noise.
